qa/common/util.sh

```diff
@@ -12,6 +12,96 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+<<<<<<< HEAD
```

Comment:
Looks like you have some unresolved merge conflicts

Reply:
This sometimes happen when save in vscode takes longer than it takes me to hit git add. Shoudlve checked